### ISSUE TITLE
Make page titles/headers more homogeneous

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-web#385]
Current situation:

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/3862206/31da3117-3df5-4a1d-847e-deed2e70df95)
![image](https://github.com/fractal-analytics-platform/fractal-web/assets/3862206/7822268f-81e7-4d8d-af56-c6679d0d53d5)
![image](https://github.com/fractal-analytics-platform/fractal-web/assets/3862206/bd9c63c5-ebde-4395-a479-822e6e59d50c)
![image](https://github.com/fractal-analytics-platform/fractal-web/assets/3862206/c40f10cb-3b72-400e-81b3-e3dcb0cfaa85)
![image](https://github.com/fractal-analytics-platform/fractal-web/assets/3862206/02977a0a-3665-45d5-a502-864171b2eccf)

**Comment by zonia3000** [comment]
Technically `h1` should be the preferred header to use for the main title of a page (also from accessibility perspective). However for me it looks a bit too heavy, so I combined it with Bootstrap `fw-light` to remove the bold. Let me know if you prefer something different.

I've also kept the breadcrumbs only on pages that have a parent page.

Ref f9f219989ae629b0f1e1543ac8cf4d0ccf9a4b23

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/ef05cbef-54b5-4383-baee-0dbd8fe9aad2)

---

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/726a2959-4925-4f0f-8c18-8186375c7090)

---

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/d7fd74f3-f07c-4f79-97c2-80cf1049dbb2)

---

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/e7eea2dd-deab-40bc-8693-c544f03d8019)

**Comment by jluethi** [comment]
Great that this becomes more uniform. Here are some minor points:

1. How does the Projects page (listing all the projects) look with this?
2. Regarding bold vs. light: no strong opinion, I'm fine with this light one.
3. Regarding breadcrumbs: The admin area doesn't necessarily need those, as admin area is a top-level link.
4. Have these new headers been applied like this everywhere? e.g. dataset pages? I actually think we could drop the title in the dataset page (similar to the workflow page), because the info is in the breadcrumbs. Same for the workflow job page

Regarding breadcrumbs vs. titles: It looks to me like we could go either/or instead of having both on some pages => top-level pages have a title, but no breadcrumbs (because those links are in the header). Pages further down the hierarchy have breadcrumbs, but no title (like workflows atm). Would that make sense?

**Comment by jluethi** [comment]
> We can remove all the titles of the pages that are directly accessible from the main menu. In that case we could also highlight the currently active section (I think we already discussed about this), to make it clearer where we are:

Oh, that's a great idea! Love it!

1-4 => perfect, let's go with that!

> Regarding the first point, what about jobs page title? It says "Jobs of user X". Is this info useful to not confuse it with the admin jobs page? Could we rename the menu link to "My Jobs" and remove the title?

I don't think that's necessary for the job page. All our users don't have access to the Admin area. Plus, the admin area is marked with the nice red bar on top, so won't be confused :)
=> Let's also drop the title on the Jobs page

**Comment by zonia3000** [comment]
![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/ebb20bfb-567d-44f6-8dc6-4131263d7c50)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/0c8924c5-b6af-4dc7-9af8-e9b678555866)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/266aef7d-ecd8-4b49-8cba-b15fc98aee81)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/10c68df1-8716-4fef-a33f-04a5b06289a1)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/f3a37f34-d8d5-4b8b-b278-7813a11b1717)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/7a305c0c-282a-42bc-87fe-a8b58806db56)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/9348e5ae-c36c-4cab-9c79-40af08586229)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/452b9081-9766-493c-b2df-0725d4fc4c46)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/72cc1f23-d9fe-43a3-a891-75e775cec19f)

![image](https://github.com/fractal-analytics-platform/fractal-web/assets/5459334/a5649e17-1736-42ba-921d-263d70c7b0eb)

**Comment by jluethi** [comment]
Looks great, thanks for the screenshots!
